src/plugins/timelion/public/directives/saved_object_finder.html

```diff
@@ -99,7 +95,7 @@
           ng-blur="finder.hitBlur($event)"
           ng-click="finder.preventClick($event)">
           <span aria-hidden="true" class="finder-type fa" ng-if="hit.icon" ng-class="hit.icon"></span>
-          <div class="kuiIcon fa-flask ng-scope" ng-if="hit.type.shouldMarkAsExperimentalInUI()"></div>
+          <icon type="'beaker'" ng-if="hit.type.shouldMarkAsExperimentalInUI()"></icon>
```

Comment:
there seems to be one last font-awesome relict , line 70-73

```
 <span
      class="fa"		      class="fa"
      ng-class="finder.isAscending ? 'fa-caret-up' : 'fa-caret-down'"		      ng-class="finder.isAscending ? 'fa-caret-up' : 'fa-caret-down'"
    >		    >
```

Reply:
Thank you @kertal, I totally missed this 🙂